query/test/src/org/labkey/test/tests/query/CrossFolderDataClassTest.java

```diff
@@ -0,0 +1,116 @@
+package org.labkey.test.tests.query;
+
+import org.junit.BeforeClass;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.pages.query.ExecuteQueryPage;
+import org.labkey.test.params.FieldDefinition;
+import org.labkey.test.params.experiment.DataClassDefinition;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.TestUser;
+import org.labkey.test.util.exp.DataClassAPIHelper;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.*;
+
+/**
+ * use this test as a place to put cross-folder dataclass testing
+ */
+@Category({})
+public class CrossFolderDataClassTest extends BaseWebDriverTest
+{
+    private static final String SUBFOLDER_A = "subA";
+    private static String SUBFOLDER_A_PATH;
+
+    @Override
+    protected void doCleanup(boolean afterTest)
+    {
+        _containerHelper.deleteProject(getProjectName(), afterTest);
+    }
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        CrossFolderDataClassTest init = (CrossFolderDataClassTest) getCurrentTest();
+
+        init.doSetup();
+    }
+
+    private void doSetup()
+    {
+        _containerHelper.createProject(getProjectName(), null);
+        _containerHelper.createSubfolder(getProjectName(), SUBFOLDER_A);
+        SUBFOLDER_A_PATH = getProjectName() + "/" + SUBFOLDER_A;
+    }
+
+    /**
+     * Issue 45644 addresses the problem where DataClass metadata wasn't available in query when querying cross-folder
```

Comment:
Issue number typo. Also, formatting the comment like "Issue xxxxx:" make TeamCity and IntelliJ link to the issue.
```suggestion
     * Issue 45664: addresses the problem where DataClass metadata wasn't available in query when querying cross-folder
```